CoYFrontend/src/app/app.component.scss

```diff
@@ -24,13 +34,12 @@
     text-decoration: none;
 }
 
-footer{
+footer {
```

Comment:
recommend to create some kind of order throughout the file like sort styles alphabetically